### ISSUE TITLE
No valid secrets error message

### ISSUE THREAD

**Issue by hvitoi** [ohwgiles/NetworkManager-f5vpn#7]
When I try to connect to the VPN I get the following error:

➜  ~ nmcli --ask c u Vivo
Error: Connection activation failed: **No valid secrets**
Hint: use 'journalctl -xe NM_CONNECTION=1efd0c06-5654-4d51-907c-d23e34aaeede + NM_DEVICE=wlp3s0' to get more details.

A prompt for entering the username and password is shown and as soon as I click OK I get the following screen
![image](https://user-images.githubusercontent.com/44772383/123021241-1aa73980-d3aa-11eb-9ec7-11ea6d8dd39e.png)

After entering user+pass I would normally have to insert an OTP.

My journalctl:
```txt
Jun 22 22:30:17 tatooine polkitd[384]: Registered Authentication Agent for unix-process:5928:179814 (system bus name :1.140 [nmcli --ask c u Vivo], object path /org/freedesktop/PolicyKit1/AuthenticationAgent, locale en_US.UTF-8)
Jun 22 22:30:17 tatooine NetworkManager[298]: <info>  [1624411817.2873] agent-manager: agent[ff9096cb1b6bb74b,:1.140/nmcli-connect/1000]: agent registered
Jun 22 22:30:17 tatooine NetworkManager[298]: <info>  [1624411817.2922] audit: op="connection-activate" uuid="1efd0c06-5654-4d51-907c-d23e34aaeede" name="Vivo" pid=5928 uid=1000 result="success"
Jun 22 22:30:17 tatooine kernel: audit: type=1111 audit(1624411817.287:238): pid=298 uid=0 auid=4294967295 ses=4294967295 msg='op=connection-activate uuid=1efd0c06-5654-4d51-907c-d23e34aaeede name="Vivo" pid=5928 uid=1000 result=success exe="/usr/bin/NetworkManager" hostname=? addr=? terminal=? res=success'
Jun 22 22:30:17 tatooine NetworkManager[298]: <info>  [1624411817.2975] vpn-connection[0x558a4fa5e140,1efd0c06-5654-4d51-907c-d23e34aaeede,"Vivo",0]: Started the VPN service, PID 5934
Jun 22 22:30:17 tatooine NetworkManager[298]: <info>  [1624411817.3116] vpn-connection[0x558a4fa5e140,1efd0c06-5654-4d51-907c-d23e34aaeede,"Vivo",0]: Saw the service appear; activating connection
Jun 22 22:30:41 tatooine gnome-shell[1196]: ../glib/gobject/gsignal.c:2732: instance '0x55ee75afc910' has no handler with id '99178'
Jun 22 22:30:41 tatooine NetworkManager[298]: <error> [1624411841.9326] vpn-connection[0x558a4fa5e140,1efd0c06-5654-4d51-907c-d23e34aaeede,"Vivo",0]: final secrets request failed to provide sufficient secrets
Jun 22 22:30:41 tatooine polkitd[384]: Unregistered Authentication Agent for unix-process:5928:179814 (system bus name :1.140, object path /org/freedesktop/PolicyKit1/AuthenticationAgent, locale en_US.UTF-8) (disconnected from bus)
Jun 22 22:30:41 tatooine NetworkManager[298]: <info>  [1624411841.9370] vpn-connection[0x558a4fa5e140,1efd0c06-5654-4d51-907c-d23e34aaeede,"Vivo",0]: VPN plugin: state changed: stopped (6)
```

Running archlinux and installed this plugin by means of the [AUR](https://aur.archlinux.org/packages/networkmanager-f5vpn/) package.
Any ideas?

**Comment by hvitoi** [comment]
By the way, I'm using Gnome Shell 40 and I have already installed network-manager-applet. No success though.

**Comment by ohwgiles** [comment]
See #6 

Looks like this is a fairly common issue, probably need to implement web login.

**Comment by hvitoi** [comment]
I see.
By what I understood the OTP input field must be in the same page as the username and password page.
In my case the OTP field appears as a separate web page after the user+pass have already been submitted.
![image](https://user-images.githubusercontent.com/44772383/123030084-91e3ca00-d3b8-11eb-9e83-5f2de171b39a.png)
That might also be the case of the other people with this similar issue

**Comment by ohwgiles** [comment]
NB: Web login implemented in da79b3f170cb
